package.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>navx</name>
+  <version>0.0.0</version>
+  <description>The navx package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="nick@todo.todo">nick</maintainer>
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
```

Comment:
Cleanup; Also, most of the code here is from Kauai labs, which released it under the MIT license, which we also use. 
There is some ambiguity (well...) since some ppl from FRC900 added code without adding a license to theirs. But we can be safe enough just doing this.